setup.py

```diff
@@ -22,24 +22,24 @@
         ],
     },
     platforms='any',
-    python_requires='>=3.5',
+    python_requires='>=3.6',
```

Comment:
Most of the requirements below have a minimum version of python 3.8 now. (All earlier versions are EOL, too.)